cityofaustin/atd-vz-data
# Add ems id to the person and primary persons table

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/14592

## Testing
**URL to test:** 
Local DB

**Steps to test:**
1. Checkout this branch, start up your local DB and apply the migrations and metadata in this branch
```bash
hasura migrate apply --envfile .env.local
hasura metadata apply --envfile .env.local
``` 
2. Nothing else really to test here, no UI changes. You can test going to the `ems__incidents` table, copying the `id` of one of the records, and then add this id to the `ems_id` field of a record in `atd_txdot_person`. Then you can see the `ems_incident` relationship for that record.


---
#### Ship list
